src/components/WorkOrders/JobRow.js

```diff
@@ -0,0 +1,41 @@
+import PropTypes from 'prop-types'
+
+const dateToStr = (date) =>
+  date.toLocaleString('en-GB', {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+  })
+
+const JobRow = ({
+  reference,
+  dateRaised,
+  lastUpdated,
+  priority,
+  propery,
+  description,
+}) => (
+  <tr className="govuk-table__row govuk-table__row--clickable govuk-body-s">
+    <td className="govuk-table__cell">{reference}</td>
+    <td className="govuk-table__cell">
+      {dateRaised ? dateToStr(dateRaised) : '—'}
```

Comment:
It would be good to have a component test for this, especially so we can test logic this like.